idx/memory/memory.go

```diff
@@ -42,19 +42,27 @@ var (
 	// metric idx.metrics_active is the number of currently known metrics in the index
 	statMetricsActive = stats.NewGauge32("idx.metrics_active")
 
-	Enabled bool
+	Enabled        bool
+	matchCacheSize int
+	tagSupport     bool
 )
 
 func ConfigSetup() {
 	memoryIdx := flag.NewFlagSet("memory-idx", flag.ExitOnError)
 	memoryIdx.BoolVar(&Enabled, "enabled", false, "")
+	memoryIdx.BoolVar(&tagSupport, "tag-support", false, "")
+	memoryIdx.IntVar(&matchCacheSize, "match-cache-size", 1000, "")
```

Comment:
please fill in help message, and add these settings, along with the same help message and default value to metrictank-sample.ini, then run the sync-configs script to apply it to the other files as well. and config-to-doc.sh